docs/api/migration-guides/qiskit-quantum-instance.mdx

```diff
@@ -138,9 +138,9 @@ primitives **expose a similar setting through their interface**:
 | Job management: `job_callback`, `max_job_retries`, `timeout`, `wait` | Does not apply | Does not apply | Sessions, callback (**) | No |
 
 (\*) For more information on error mitigation and setting options on Qiskit Runtime Primitives, visit
-[this link](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.options.Options.html#qiskit_ibm_runtime.options.Options).
+[Advanced Runtime Options](../../run/advanced-runtime-options).
 
-(\*\*) For more information on Runtime sessions, visit [this how-to](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/how_to/run_session.html).
+(\*\*) For more information on Runtime sessions, visit [About Sessions](../../run/sessions).
```

Comment:
@abbycross did I choose the correct pages?

Reply:
I think so! There's also the error mitigation page https://docs.quantum-computing.ibm.com/run/configure-error-mitigation but the advanced runtime options page points to it as part of Next steps, so maybe no need to call out on its own?